tests/unit/test__helpers.py

```diff
@@ -233,6 +233,66 @@ def test_w_unknown_type(self):
         with self.assertRaises(ValueError):
             self._callFUT(object())
 
+    def test_w_numeric_precision_and_scale_valid(self):
+        import decimal
+        from google.protobuf.struct_pb2 import Value
+
+        cases = [
+            decimal.Decimal("42"),
+            decimal.Decimal("9.9999999999999999999999999999999999999E+28"),
+            decimal.Decimal("-9.9999999999999999999999999999999999999E+28"),
+            decimal.Decimal("99999999999999999999999999999.999999999"),
+            decimal.Decimal("1E+28"),
+            decimal.Decimal("1E-9"),
+        ]
+        for value in cases:
+            with self.subTest(value=value):
+                value = decimal.Decimal("9999999999999999999999999999.999999999")
```

Comment:
I don't think this line is needed. The value should be coming from the cases.

Reply:
changed to read from cases.